shell/platform/windows/platform_handler.h

```diff
@@ -22,6 +22,11 @@ namespace flutter {
 class FlutterWindowsEngine;
 class ScopedClipboardInterface;
 
+enum class ExitType {
```

Comment:
Please add a comment that this must be kept in sync with `platform_handler.cc`'s `exit_type_names`